frontend/src/components/Map/index.js

```diff
@@ -23,617 +32,640 @@ export default function Map() {
+      state: "California",
+      fips: "06077",
+      date: "2020-05-06",
+      cases: 579,
+    },
+    {
+      deaths: 1,
+      county: "San Luis Obispo",
+      state: "California",
+      fips: "06079",
+      date: "2020-05-06",
+      cases: 208,
+    },
+    {
+      deaths: 56,
+      county: "San Mateo",
+      state: "California",
+      fips: "06081",
+      date: "2020-05-06",
+      cases: 1341,
+    },
+    {
+      deaths: 10,
+      county: "Santa Barbara",
+      state: "California",
+      fips: "06083",
+      date: "2020-05-06",
+      cases: 613,
+    },
+    {
+      deaths: 122,
+      county: "Santa Clara",
+      state: "California",
+      fips: "06085",
+      date: "2020-05-06",
+      cases: 2268,
+    },
+    {
+      deaths: 2,
+      county: "Santa Cruz",
+      state: "California",
+      fips: "06087",
+      date: "2020-05-06",
+      cases: 138,
+    },
+    {
+      deaths: 4,
+      county: "Shasta",
+      state: "California",
+      fips: "06089",
+      date: "2020-05-06",
+      cases: 31,
+    },
+    {
+      deaths: 0,
+      county: "Siskiyou",
+      state: "California",
+      fips: "06093",
+      date: "2020-05-06",
+      cases: 5,
+    },
+    {
+      deaths: 7,
+      county: "Solano",
+      state: "California",
+      fips: "06095",
+      date: "2020-05-06",
+      cases: 342,
+    },
+    {
+      deaths: 4,
+      county: "Sonoma",
+      state: "California",
+      fips: "06097",
+      date: "2020-05-06",
+      cases: 286,
+    },
+    {
+      deaths: 17,
+      county: "Stanislaus",
+      state: "California",
+      fips: "06099",
+      date: "2020-05-06",
+      cases: 461,
+    },
+    {
+      deaths: 2,
+      county: "Sutter",
+      state: "California",
+      fips: "06101",
+      date: "2020-05-06",
+      cases: 31,
+    },
+    {
+      deaths: 1,
+      county: "Tehama",
+      state: "California",
+      fips: "06103",
+      date: "2020-05-06",
+      cases: 1,
+    },
+    {
+      deaths: 41,
+      county: "Tulare",
+      state: "California",
+      fips: "06107",
+      date: "2020-05-06",
+      cases: 916,
+    },
+    {
+      deaths: 0,
+      county: "Tuolumne",
+      state: "California",
+      fips: "06109",
+      date: "2020-05-06",
+      cases: 2,
+    },
+    {
+      deaths: 19,
+      county: "Ventura",
+      state: "California",
+      fips: "06111",
+      date: "2020-05-06",
+      cases: 608,
+    },
+    {
+      deaths: 20,
+      county: "Yolo",
+      state: "California",
+      fips: "06113",
+      date: "2020-05-06",
+      cases: 172,
+    },
+    {
+      deaths: 1,
+      county: "Yuba",
+      state: "California",
+      fips: "06115",
+      date: "2020-05-06",
+      cases: 19,
+    },
+  ];
 
-      var colorScale = chroma
-        .scale(["black", "red"])
-        .padding(0.15)
-        .domain(values, "q", 5);
+  var values = countyData.map((county) => county.deaths); //get all the death data points
```

Comment:
`var`s are global - all other components within the app can see this variable. This should be a `const`, or a `let` if it's to be modified in runtime.

This applies to all other `var`s except `map`, which I think actually needs to be global in order for mapbox to work.

Reply:
got it! This is part of the already-existing-code, but good comment for the future, will keep in mind 

All this will be removed since is part of the `addUSCountyLayer` function that is not being used